x-pack/plugins/cases/public/components/create/description.test.tsx

```diff
@@ -7,113 +7,135 @@
+            tags: [],
+          }}
+        >
+          <Description {...defaultProps} />
+        </FormTestComponent>
+      );
+
+      const description = await screen.findByTestId('euiMarkdownEditorTextArea');
+
+      userEvent.clear(description);
+      userEvent.paste(description, 'My new description');
+
+      userEvent.click(screen.getByText('Submit'));
+
+      await waitFor(() => {
+        expect(onSubmit).toBeCalledWith({ description: 'My new description' }, true);
+      });
+    });
 
-  it('it renders', async () => {
-    appMockRender.render(
-      <MockHookWrapperComponent>
-        <Description {...defaultProps} />
-      </MockHookWrapperComponent>
-    );
+    it('shows an error when description is empty', async () => {
+      appMockRender.render(
+        <FormTestComponent
+          onSubmit={onSubmit}
+          formDefaultValue={{
+            description: 'default description',
+            title: 'Default title',
+            tags: [],
+          }}
+          schema={{ description: schema.description } as FormSchema}
+        >
+          <Description {...defaultProps} />
+        </FormTestComponent>
+      );
 
-    expect(screen.getByTestId('caseDescription')).toBeInTheDocument();
-  });
+      const description = await screen.findByTestId('euiMarkdownEditorTextArea');
 
-  it('it changes the description', async () => {
-    appMockRender.render(
-      <MockHookWrapperComponent>
-        <Description {...defaultProps} />
-      </MockHookWrapperComponent>
-    );
+      userEvent.clear(description);
 
-    const description = screen.getByTestId('euiMarkdownEditorTextArea');
+      userEvent.paste(description, '  ');
 
-    userEvent.type(
-      description,
-      `${specialChars.selectAll}${specialChars.delete}My new description`
-    );
+      userEvent.click(screen.getByText('Submit'));
 
-    await waitFor(() => {
-      expect(globalForm.getFormData()).toEqual({ description: 'My new description' });
-    });
-  });
-
-  it('shows an error when description is empty', async () => {
-    appMockRender.render(
-      <MockHookWrapperComponent>
-        <Description {...defaultProps} />
-      </MockHookWrapperComponent>
-    );
+      await waitFor(() => {
+        expect(onSubmit).toBeCalledWith({}, false);
+      });
 
-    const description = screen.getByTestId('euiMarkdownEditorTextArea');
+      expect(await screen.findByText('A description is required.')).toBeInTheDocument();
+    });
 
-    userEvent.clear(description);
-    userEvent.type(description, '  ');
+    it('shows an error when description is too long', async () => {
+      const longDescription = 'a'.repeat(MAX_DESCRIPTION_LENGTH + 1);
 
-    await waitFor(() => {
-      expect(screen.getByText('A description is required.')).toBeInTheDocument();
-    });
-  });
+      appMockRender.render(
+        <FormTestComponent
+          onSubmit={onSubmit}
+          formDefaultValue={{
+            description: 'default description',
```

Comment:
Maybe without this default value, you could remove the `clear` on line 125 and the tests would be even faster ⚡

(or not, I'm not sure :D but maybe worth a try, `userEvent` is usually slow)